foosball/games/models.py

```diff
@@ -25,7 +24,7 @@ class Game(TimeStampedModel):
     )
 
     played_at = models.DateTimeField(
-        default=datetime.datetime.now,
+        default=timezone.now,
```

Comment:
That sounds like a good idea at first, but we want this field to be edited at the form level (games may not be stored at the time they were played). When `auto_add` or `auto_now_add` is set to `True`, Django forcefully overrides the values of `editable` and `blank` like this:

```python
if auto_now or auto_now_add:
    kwargs['editable'] = False
    kwargs['blank'] = True
```

Since we're setting the default value for the model (and hence the form) I think it's perfectly reasonable to use `default` here.